### PR TITLE
[11.0][IMP] message_auto_subscribe_notify_own: Allow to select models…

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
… to be notified

CC @jbeficent

### CODE REVIEW

**Comment by chienandalu** [comment]
@hveficent Is it WIP?

**Comment by HviorForgeFlow** [comment]
@chienandalu, more or less it is. I can't figure out how to implement unit testing. This approach is failing on Test shutdown.

Any idea to move it forward

File: message_auto_subscribe_notify_own/tests/test_fake_model.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# Copyright 2019 Aleph Objects, Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class FakeModelToTest(models.Model):
+    """A Fake model to Test."""
+
+    _name = 'fake.model'
```
Comment (MiquelRForgeFlow) [inline message_auto_subscribe_notify_own/tests/test_fake_model.py]: `message_auto_subscribe_notify_own.test` should be a better name. This way we can identify the table from which module comes from.

**Review by MiquelRForgeFlow — APPROVED** [review]

File: message_auto_subscribe_notify_own/tests/test_fake_model.py (hunk 1 above)
Comment (HviorForgeFlow) [reply message_auto_subscribe_notify_own/tests/test_fake_model.py]: Ok! it makes sense, changed

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request closed without merging**